MMM-TeslaFi.js

```diff
@@ -281,5 +307,25 @@ Module.register('MMM-TeslaFi', {
 		} else {
 			return this.numberFormat(valueMiles) + " miles";
 		}
+	},
+
+	// Converts given speed (assumes miles input) to configured output with approprate units appened
+	convertSpeed: function(valueMiles) {
+		if(this.config.unitDistance==="km") {
+			return this.numberFormat(valueMiles * 1.60934) + " Km/h"
+		} else {
+			return this.numberFormat(valueMiles) + " Mph"
```

Comment:
As above, could you please change this to "mph"?

Reply:
Just pushed this. done.
Good catch!